src/System.Management.Automation/engine/CommandProcessorBase.cs

```diff
@@ -682,11 +675,9 @@ public override string ToString()
         /// This default implementation reads the next pipeline object and sets
         /// it as the CurrentPipelineObject in the InternalCommand.
         /// </summary>
-        ///
         /// <returns>
         /// True if read succeeds.
         /// </returns>
-        ///
         /// does not throw
```

Comment:
Can we move this to inside the \<summary/\>?

Reply:
Fixed.